### DISCUSSION TITLE
Welcome to mwClipboard Discussions!

### DISCUSSION THREAD

**Discussion by mikeyww** [mikeyww/mwClipboard#1] [Announcements]

**Comment by JC055** [comment]
Hi, I just downloaded this, and it is working great, but I am having trouble editing the clipboard. I am not sure what I'm doing wrong. How is that supposed to work?

Additionally, two possible suggestions to consider: 
1) It'd be helpful if we could delete clips directly from the GUI, 
2) it would also be nice if we could rearrange or pin certain clips for easier access. 

I understand this may not be as easy to setup as it sounds, but I just wanted to put that out there.
Thanks!

**Comment by mikeyww** [comment]
Thank you for the comments. 

You are having trouble editing the clipboard because this program does not enable any such editing. The editing menu option opens the clips _file_ for editing, viewing, or searching in the default text editor, but it does not directly alter the clipboard. All text clips are saved in a plain text file, but this is _simply like a log_ of what was in the clipboard. It is not the clipboard itself.

I appreciate the suggestions as well. The program is not meant to have "advanced" management features as some other programs do have. It is not meant to reorganize clips, delete clips, etc., but simply to keep the history so that the clips can be used again. You can do a simple search for the clips, either in the GUI or the text file. In addition, a clip can be moved to the top simply by right-clicking it, which copies it to the clipboard again.

Enjoy!

**Reply by JC055** [reply]
Got it. I appreciate your response!
